src/Refactoring-Changes/RBClassRepackagingChange.class.st

```diff
@@ -25,7 +25,7 @@ RBClassRepackagingChange class >> repackage: aClass in: aPackage tag: aTag [
 { #category : 'converting' }
 RBClassRepackagingChange >> asUndoOperation [
 
-	^ self class repackage: self changeClass in: self changeClass packageName tag: self changeClass tagName
+	^ self class repackage: self changeClass in: self changeClass packageName tag: self changeClass packageName
```

Comment:
I'm not sure about this change. It seems to me that we are losing an information using the package name as the tag name

Reply:
Thanks for catching this, I had this in the first version, but I guess I lost it along the way.